snowplow-tracker/build.gradle

```diff
@@ -76,13 +76,16 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:support-annotations:28.0.0'
-    implementation "android.arch.lifecycle:extensions:$project.archLifecycleVersion"
+    implementation 'androidx.annotation:annotation:1.1.0'
+    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.squareup.okhttp3:okhttp:3.12.12'
+    implementation 'com.squareup.okhttp3:okhttp:4.0.0'
     // test
-    androidTestImplementation 'com.google.android.gms:play-services-analytics:16.0.1'
-    androidTestImplementation 'com.squareup.okhttp3:mockwebserver:3.12.12'
+    testImplementation 'junit:junit:4.13.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    androidTestImplementation 'com.google.android.gms:play-services-analytics:17.0.0'
+    androidTestImplementation 'com.squareup.okhttp3:mockwebserver:4.0.0'
```

Comment:
And here

Reply:
same answer above. I'll give a second look to this in the next beta version.